index.php

```diff
@@ -0,0 +1,311 @@
+<?php
+    if(!$_GET["page"]){
+        header("location:index.php?page=home");
+    }
+?>
+
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="UTF-8">
+        <title>PT. Pindad (Persero) - Home</title>
+        <link rel="icon" href="img/favicon.ico" type="image/x-icon">
+        <?php
+            echo "<link rel='stylesheet' href='css/".$_GET["page"].".css'>";
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Detected usage of a non-sanitized input variable: $_GET["page"]](https://app.codacy.com/manual/anamnafiul99/pindad-clone/pullRequest?prid=5431765)